makePoplogCommander.sh

```diff
@@ -144,6 +232,13 @@ poplog env [OPTION]... [-] [NAME=VALUE]... [COMMAND [ARG]...]
 
         % poplog exec env [OPTION]... [-] [NAME=VALUE]... [COMMAND [ARG]...]
 
+poplog [NAME=VALUE]... [COMMAND [ARG]...]
+    Adds/modifies environment variables in the Poplog environment and
```

Comment:
"Adds/modifies environment variables in the Poplog environment" -> "Adds/modifies environment variables in the Poplog environment for the duration of the session running COMMAND" (or something along those lines)

Reply:
OK!